source/win-main/DocumentTabs.vue

```diff
@@ -316,11 +316,19 @@ export default {
 <style lang="less">
 @tabbar-height: 30px;
 
+body {
+  &.dark {
+    div #tab-container {
+      border-top: 1px solid rgb(30, 30, 30);
+    }
+  }
+}
+
 body div#tab-container {
   width: 100%;
   height: 30px;
   background-color: rgb(215, 215, 215);
-  border-bottom: 1px solid grey;
+  border-top: 1px solid grey;
```

Comment:
Since the macOS and Windows styles don't have a border-top (at least I think so …?) we have to reset these in both the Windows and macOS styles so it doesn't break the design.